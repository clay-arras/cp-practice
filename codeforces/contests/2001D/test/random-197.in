1
9
17 9 6 13 11 13 3 13 13
