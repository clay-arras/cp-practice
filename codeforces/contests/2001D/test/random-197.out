6
17 9 6 11 13 3
